pkg/naming/labels.go

```diff
@@ -30,6 +30,15 @@ const (
 	LabelPerconaRestoreJobName     = perconaPrefix + "restore-job-name"
 )
 
+func GetLabelBackupType(cr *api.PerconaXtraDBCluster) string {
+	labelKeyBackupType := "type"
+	if cr.CompareVersionWith("1.16.0") >= 0 {
+		labelKeyBackupType = LabelPerconaBackupType
+	}
+
+	return labelKeyBackupType
+}
+
```

Comment:
small suggestion (not critical)
can we invert logic to 
```
labelKeyBackupType := LabelPerconaBackupType
	if cr.CompareVersionWith("1.16.0") < 0 {
		labelKeyBackupType = "type"
	}
```
In this case we can easier refactor when type is not supported anymore.